pyaedt/misc/spisim_com_configuration_files/com_parameters.py

```diff
@@ -0,0 +1,375 @@
+                fp.write("# {0}: {0}\n".format(kw.upper()))
+                fp.write("{} = {}\n".format(kw.upper(), v))
+        return True
+
+    @pyaedt_function_handler
+    def load_spisim_cfg(self, file_path):
+        """Load a SPIsim configuration file.
+
+        Parameters
+        ----------
+        file_path: str
+            Path of the configuration file.
+
+        Returns
+        -------
+        bool
+            ``True`` when successful, ``False`` when failed.
+        """
+        reverse_map = {j: i for i, j in spimsim_matlab_keywords_mapping.items()}
+
+        with open(file_path, "r") as fp:
+            lines = fp.readlines()
+            for line in lines:
+                if not line.startswith("#") and "=" in line:
+                    split_line = [i.strip() for i in line.split("=")]
+                    kw, value = split_line
+                    if kw in reverse_map:  # Get Matlab keyword
+                        kw = reverse_map[kw]
+                    self.set_parameter(kw, value)
+        return
+
+
+class COMParametersVer3p4(COMParameters):
+    """Manages COM parameters of version 3.4."""
+
+    def __init__(self, standard="50GAUI-1-C2C"):
+        super().__init__(standard)
+
+    @pyaedt_function_handler
+    def _init(self):
+        """Initialize COM parameters."""
+        self.table_93a1.update(
+            {
+                "f_b": "",
+                "f_min": "",
+                "Delta_f": "",
+                "C_d": "",
+                "L_s": "",
+                "C_b": "",
+                "z_p select": "",
+                "z_p (TX)": "",
+                "z_p (NEXT)": "",
+                "z_p (FEXT)": "",
+                "z_p (RX)": "",
+                "C_p": "",
+                "R_0": "",
+                "R_d": "",
+                "A_v": "",
+                "A_fe": "",
+                "A_ne": "",
+                "AC_CM_RMS": "",
+                "L": "",
+                "M": "",
+            }
+        )
+        self.filter_and_eq.update(
+            {
+                "f_r": "",
+                "c(0)": "",
+                "c(-1)": "",
+                "c(-2)": "",
+                "c(-3)": "",
+                "c(1)": "",
+                "N_b": "",
+                "b_max(1)": "",
+                "b_max(2..N_b)": "",
+                "b_min(1)": "",
+                "b_min(2..N_b)": "",
+                "g_DC": "",
+                "f_z": "",
+                "f_p1": "",
+                "f_p2": "",
+                "g_DC_HP": "",
+                "f_HP_PZ": "",
+            }
+        )
+        self.io_control.update(
+            {
+                # "DIAGNOSTICS": "",
+                # "DISPLAY_WINDOW": "",
+                # "CSV_REPORT": "",
+                "RESULT_DIR": "",
+                # "SAVE_FIGURES": "",
+                "Port Order": "",
+                "RUNTAG": "",
+                # "COM_CONTRIBUTION": "",
```

Comment:
Keep this comments ?

Reply:
keep them here for now. I am not sure if they will be used in the later enhancement.